### PR TITLE
Fix custom context in popout mode

### PR DESCRIPTION
**Pull request by akhiloct1510**
Fixing the custom context data is not setup in V2 for popout mode (Unauthenticated).

Un-Auth Poput Mode V2 

image

image

Authenticated Popout V2. Context being passed using JWT & SDK method. The JWT context shows up while the SDK method is not (expected behavior)

image

image

Un-Auth - Embedded Mode V2
image

image

Authenticated Embedded V2 Context being passed using JWT & SDK method. The JWT context shows up while the SDK method is not (expected behavior)

image

image

### CODE REVIEW

File: chat-widget/src/components/livechatwidget/common/startChat.ts [hunk 1]
```diff
@@ -263,12 +263,20 @@ const setCustomContextParams = (props?: ILiveChatWidgetProps) => {
...
             Event: TelemetryEvent.SetCustomContext,
-            Description: "Setting custom context for unauthenicated chat"
+            Description: "Setting custom context for unauthenticated chat"
         });
 
         optionalParams = Object.assign({}, optionalParams, {
             customContext: persistedState?.domainStates?.customContext
         });
+    } else
+    {
```
Comment (charliewang95) [inline chat-widget/src/components/livechatwidget/common/startChat.ts:273]: nit: spacing

Comment (akhiloct1510) [reply chat-widget/src/components/livechatwidget/common/startChat.ts]: Fixed

File: chat-widget/src/components/livechatwidget/common/startChat.ts [hunk 2]
```diff
@@ -263,12 +263,19 @@ const setCustomContextParams = (props?: ILiveChatWidgetProps) => {
...
         TelemetryHelper.logLoadingEvent(LogLevel.INFO, {
             Event: TelemetryEvent.SetCustomContext,
-            Description: "Setting custom context for unauthenicated chat"
+            Description: "Setting custom context for unauthenticated chat"
         });
 
         optionalParams = Object.assign({}, optionalParams, {
             customContext: persistedState?.domainStates?.customContext
         });
+    } else {
```
Comment (charliewang95) [inline chat-widget/src/components/livechatwidget/common/startChat.ts:272]: Please verify if non-popout still work with this change

**Review by charliewang95 — APPROVED** [review]

File: chat-widget/src/components/livechatwidget/common/startChat.ts [hunk 3]
```diff
@@ -333,4 +340,33 @@ const checkIfConversationStillValid = async (chatSDK: any, dispatch: Dispatch<IL
         return false;
     }
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const getInitContextParamsForPopout = async (): Promise<any> => {
```
Comment (charliewang95) [inline chat-widget/src/components/livechatwidget/common/startChat.ts:345]: Update ChangeLog please

File: chat-widget/src/components/livechatwidget/common/startChat.ts (hunk 2 above)
Comment (akhiloct1510) [reply chat-widget/src/components/livechatwidget/common/startChat.ts:272]: It is working. I have attached test evidences for Auth/UnAuth with embedded & PopOut mode

**Pull request merged**